src/main/java/trackbot/TrackBotStorage.java

```diff
@@ -0,0 +1,90 @@
+package trackbot;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
```

Comment:
I put a blank line to split up the java.io and java.util packages but that's just a personal preference.